tests/integration/suite/test_globaldns.py

```diff
@@ -261,3 +261,24 @@ def test_cloudflare_provider_proxy_setting(admin_mc, remove_resource):
     assert gdns_provider.cloudflareProviderConfig.proxySetting is True
 
     remove_resource(globaldns_provider)
+
+
+def test_dns_fqdn_hostname(admin_mc):
+    client = admin_mc.client
+    provider_name = random_str()
+    access = random_str()
+    secret = random_str()
+    globaldns_provider = \
+        client.create_global_dns_provider(
+                                         name=provider_name,
+                                         rootDomain="example.com",
+                                         route53ProviderConfig={
+                                             'accessKey': access,
+                                             'secretKey': secret})
+
```

Comment:
Can we use `remove_resource(globaldns_provider)` after creation so this gets deleted even if test fails before `client.delete(globaldns_provider)` is called?

Reply:
done